bats/tests/containers/auto-start.bats

```diff
@@ -0,0 +1,113 @@
+load '../helpers/load'
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'Start up Rancher Desktop' {
+    start_application
+}
+
+get_json_key() {
+    local json_key=$1
+    rdctl list-settings | jq -r "${json_key}" 3>&-
```

Comment:
Why do you need `3>&-` here?

Also maybe split the pipe into 2 separate commands:

```suggestion
    run rdctl list-settings
    assert_success
    jq -r "${json_key}" <<<"${output}"
```